src/components/Login/Login.jsx

```diff
@@ -23,7 +23,12 @@ export class Login extends React.Component {
         })
         .then(response => {
             if (response.status === 200) {
+                console.log("log in successful, attempt to connect to socket");
+                this.setState ({
+                    loggedIn: true
+                })
                 window.location = '/dashboard';
```

Comment:
This was from another PR but I'll pop it here again. Have a look into `history.push`.